### PR TITLE
Remove explicit PNet resizes from MTCNN graph

### PR DESCRIPTION
**Pull request by dbudniko**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

force_builders=Custom,Custom Win,Custom Mac
build_gapi_standalone:Linux x64=ade-0.1.1f
build_gapi_standalone:Win64=ade-0.1.1f
build_gapi_standalone:Mac=ade-0.1.1f
build_gapi_standalone:Linux x64 Debug=ade-0.1.1f

build_image:Custom=centos:7
buildworker:Custom=linux-1
build_gapi_standalone:Custom=ade-0.1.1f

Xbuild_image:Custom=ubuntu-openvino-2020.3.0:16.04
build_image:Custom Win=openvino-2021.3.0
build_image:Custom Mac=openvino-2021.3.0

test_modules:Custom=gapi,python2,python3,java
test_modules:Custom Win=gapi,python2,python3,java
test_modules:Custom Mac=gapi,python2,python3,java

buildworker:Custom=linux-1
// disabled due high memory usage: test_opencl:Custom=ON
test_opencl:Custom=OFF
test_bigdata:Custom=1
test_filter:Custom=*

### CODE REVIEW

File: modules/gapi/samples/face_detection_mtcnn.cpp [hunk 1]
```diff
@@ -598,19 +598,15 @@ int main(int argc, char* argv[]) {
...
+    std::tie(regressions[0], scores[0]) = run_mtcnn_p(in_transposedRGB, get_pnet_level_name(level_size[0]));
     cv::GArray<custom::Face> faces0 = custom::BuildFaces::on(scores[0], regressions[0], static_cast<float>(scales[0]), conf_thresh_p);
     cv::GArray<custom::Face> final_p_faces_for_bb2squares = custom::ApplyRegression::on(faces0, true);
     cv::GArray<custom::Face> final_faces_pnet0 = custom::BBoxesToSquares::on(final_p_faces_for_bb2squares);
     total_faces[0] = custom::RunNMS::on(final_faces_pnet0, 0.5f, false);
     //The rest PNet pyramid layers to accumlate all layers result in total_faces[PYRAMID_LEVELS - 1]]
     for (int i = 1; i < pyramid_levels; ++i)
     {
-        currentSize = cv::Size(level_size[i].height, level_size[i].width);
-        in_resized[i] = cv::gapi::resize(in_transposedRGB, currentSize);
```
Comment (rgarnov) [inline modules/gapi/samples/face_detection_mtcnn.cpp:612]: As far as I can see `in_resized` became unused and can be removed

**Review by rgarnov — CHANGES_REQUESTED** [review]

File: modules/gapi/samples/face_detection_mtcnn.cpp (hunk 1 above)
Comment (dbudniko) [reply modules/gapi/samples/face_detection_mtcnn.cpp:612]: Done

**Review by TolyaTalamanov — COMMENTED** [review]
@dbudnikoWhat is the motivation for removing resize?

**Comment by dbudniko** [comment]
@TolyaTalamanov because resize PP is done on IE level

**Review by TolyaTalamanov — APPROVED** [review]
LGTM 👍

**Pull request merged**